extensions/flac/src/main/jni/include/flac_parser.h

```diff
@@ -30,6 +30,17 @@
 
 typedef int status_t;
 
+typedef struct {
```

Comment:
Not sure if it's the same, but elsewhere in jni code we're using style like:

```
struct FlacPicture {
  int type;
  ....
}
```

with the name starting with a capital letter. If it's the same, could we do it that way for consistency? Ignore if not :).

Reply:
Done